src/main/java/org/jfrog/hudson/npm/NpmInstallCallable.java

```diff
@@ -0,0 +1,43 @@
+package org.jfrog.hudson.npm;
+
+import hudson.remoting.VirtualChannel;
+import jenkins.MasterToSlaveFileCallable;
+import org.apache.commons.lang3.StringUtils;
+import org.jfrog.build.api.Build;
+import org.jfrog.build.api.util.Log;
+import org.jfrog.build.extractor.clientConfiguration.ArtifactoryDependenciesClientBuilder;
+import org.jfrog.build.extractor.npm.extractor.NpmInstall;
+import org.jfrog.hudson.pipeline.Utils;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.Objects;
+
+/**
+ * Created by Yahav Itzhak on 25 Nov 2018.
+ */
+public class NpmInstallCallable extends MasterToSlaveFileCallable<Build> {
+
+    private ArtifactoryDependenciesClientBuilder dependenciesClientBuilder;
+    private String resolutionRepository; // Artifactory repository to download dependencies.
+    private String executablePath; // npm executable path. Can be empty.
+    private String args; // npm args.
+    private String path; // Path to package.json or path to the directory that contains package.json.
+    private Log logger;
+
```

Comment:
I suggest we add the member variables description above the variables, so that it is easier to read.